script/DeployLeveragedEthTestnet.s.sol

```diff
@@ -1,19 +1,128 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity ^0.8.13;
 
+import "forge-std/console2.sol";
+import "forge-std/Test.sol";
+import "forge-std/StdStorage.sol";
+
 import {DeployLeveragedEth} from "./base/DeployLeveragedEth.sol";
 import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 import {MockWETH} from "../test/mocks/MockWETH.sol";
 import {MockUSDC} from "../test/mocks/MockUSDC.sol";
-import {GoerliConstants as C} from "../src/lib/GoerliConstants.sol";
+import {Constants as C} from "../src/lib/Constants.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {sc4626} from "../src/sc4626.sol";
+import {scWETH} from "../src/steth/scWETH.sol";
+import {scUSDC} from "../src/steth/scUSDC.sol";
 
-contract DeployScript is DeployLeveragedEth {
-    function deployMockTokens() internal override {
-        weth = MockWETH(payable(C.WETH));
-        usdc = MockUSDC(C.USDC);
-    }
+contract DeployScript is DeployLeveragedEth, Test {
```

Comment:
Also can you add a title explaining the function of the script and the command required to run this script.